torch/csrc/distributed/c10d/TCPStoreLibUvBackend.cpp

```diff
@@ -288,20 +288,18 @@ class UvTcpServer : public UvTcpSocket {
 
       uv_res =
           uv_listen(res->unsafeGetStream(), DEFAULT_BACKLOG, on_new_connection);
-      TORCH_CHECK(
-          uv_res == 0,
-          "The server socket has failed to listen on any local network address. ",
-          "port: ",
-          port,
-          ", useIpv6: ",
-          useIpv6,
-          ", code: ",
-          uv_res,
-          ", name: ",
-          uv_err_name(uv_res),
-          ", message: ",
-          uv_strerror(uv_res));
-
+      if (uv_res != 0) {
```

Comment:
nice, maybe we can (I mean I can do it) to replace all these `TORCH_CHECK` with `C10D_THROW_ERROR` so that errors can be surfaces easier for debuggability.

Reply:
I'll take a look and see which ones I can easily fix